<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaboratoryView</class>
 <widget class="QMainWindow" name="LaboratoryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LaboratoryView</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(47, 0, 74);
	color: white;
}

QLineEdit {
	background-color: rgb(62, 1, 97);
	color: white;
	border: 0px
}

QMessageBox {
	background-color: rgb(47, 0, 74);
	color: white;
}

QToolTip {
	color: white;
	background-color: rgb(47, 0, 74);
	border: 0px;
}

QSpinBox {
	background-color: rgb(62, 1, 97);
	color: white;
	border: 0px;
}

QPushButton {
background-color: rgb(62, 1, 97);
color: white;
}

QPushButton::hover {
	background-color: rgb(72, 1, 112);
}

QLabel {
	background-color: rgb(47, 0, 74);
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>11</y>
      <width>581</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSortedStatisticPushButton">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>60</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сортированная выборка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadStatisticPushButton">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить выборку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="intervalsBuildPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Разбить выборку </string>
    </property>
   </widget>
   <widget class="QPushButton" name="getStatisticIntervalsPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Получить интервалы</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>389</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Количество интервалов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="intervals_count">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>310</y>
      <width>194</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дисперсия выборки D(X):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statistic_dispersion">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_const_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>350</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отклонение выборки σ(X):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statistic_deviation">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_const_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>270</y>
      <width>227</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Мат. ожидание выборки M(X):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statistic_mean">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNormalDistributionGraphPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нормальное распределение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма выборки Σ(X):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statistic_sum">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelconst_1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>110</y>
      <width>157</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Размер выборки (N):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statistic_size">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statistic_min_value">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>150</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Мин. значение (Xmin):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statistic_max_value">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>190</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Макс. значение (Xmax):</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить значение:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="erase_value_line_edit">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eraseStatisticValuePushButton">
       <property name="font">
        <font>
         <family>Microsoft New Tai Lue</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
